### PR TITLE
Indented and cleaned code

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #45 
Indented
Updated head.html, side-bar.html, about.html

### CODE REVIEW

File: _includes/side_bar.html [hunk 1]
```diff
@@ -1,18 +1,16 @@
-	</div>
-	
-	<div id="side-bar">
+	</div>	
+	 <div id="side-bar">
```
Comment (anshumanv) [inline _includes/side_bar.html]: Not indented properly.

File: _includes/side_bar.html [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
 		<h1>Platinum Sponsor</h1>
 		<a href="http://google.com/" target="_blank"><img alt="sponsor logo" class="sponsor" src="images/side_images/google_logo.png" /></a>
 		<a href="http://www.swedenabroad.com/en-GB/Embassies/Phnom-Penh/" target="_blank"><img alt="sponsor logo" class="sponsor" src="images/side_images/embassyofsweden.png"></a>
-
+
```
Comment (anshumanv) [inline _includes/side_bar.html]: Why this spacing?

**Review by anshumanv — COMMENTED** [review]
Check comments.

**Comment by arjunpandey2003** [comment]
@anshumanv made the required changes

**Review by anshumanv — APPROVED** [review]
LGTM

**Comment by arjunpandey2003** [comment]
@anshumanv thanks

**Pull request merged**